src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -605,7 +605,7 @@ private void createAndSetStatistics()
                 int max = ((AbstractBuildingWorker.View) building).getMaxInhabitants();
                 int workers = ((AbstractBuildingWorker.View) building).getWorkerId().size();
 
-                String jobName = ((AbstractBuildingWorker.View) building).getJobName().toLowerCase(Locale.ENGLISH);
+                String jobName = ((AbstractBuildingWorker.View) building).building.getJobDisplayName().toLowerCase(Locale.ENGLISH);
```

Comment:
Close, not quite. Remove the building. out of this, since the code before that is the conceptual equivlent. You just need a direct replacement of getJobName with getJobDisplayName

Reply:
I think I fixed it

Reply:
Yeah, that looks better. Now you can undo the other changes, and verify that it's doing what you want. :)

Reply:
I think I already undid the changes to the buildings. How do I test/verify it?